src/blog/_posts/2018-09-26-opera-56.md

```diff
@@ -0,0 +1,155 @@
+---
+title: What's new in Chromium 69 and Opera 56
+authors:
+- daniel-bratell
+intro: 'Opera 56 (based on Chromium 69) for [Mac, Windows, Linux](https://www.opera.com/computer) is out! To find out what’s new for users, see our [Desktop blog post](https://blogs.opera.com/desktop/2018/09/opera-56-stable). Here’s what it means for web developers.'
+tags:
+- html
+- css
+- dom
+- javascript
+- opera
+license: cc-by-3.0
+---
+
+Opera 56 (based on Chromium 69) for [Mac, Windows, Linux](https://www.opera.com/computer) is out! To
+find out what’s new for users, see our
+[Desktop blog post](https://blogs.opera.com/desktop/2018/09/opera-56-stable).
+Here’s what it means for web developers.
+
+
+## CSS
+
+### [Conic gradients](https://leaverou.github.io/conic-gradient/)
+
+The web already has linear gradients where the colour depends on the
+distance to a line, and radial gradients where the colour depends on
+the distance to a point. Now it also gets conic gradients where the
+colour depends on the angular distance to a line through the new
+[`conic-gradient`](https://www.w3.org/TR/css-images-4/#conic-gradients)
+property.
+
+Example:
+
+<div style="border: 3px solid black; background: conic-gradient(red, yellow, green, red); width: 400px; height: 400px;"></div>
+
+### Logical margin, paddings and border properties
+
+Blink, the Chromium/Opera browser engine, has gotten some [new CSS
+properties](https://drafts.csswg.org/css-logical-1/#box) that adapt to
+text direction. So `padding-inline-start` will apply to either left or
+right depending on what side you start reading the start. Other
```

Comment:
Drop the 'the start' at the end ("start reading the start".)

Reply:
Fixed, thasnks!